app/api/routes/resource_retrieval.py

```diff
@@ -42,7 +43,19 @@ def get_resources():
     updated_after = request.args.get('updated_after')
     free = request.args.get('free')
 
+    apikey = request.headers.get('x-apikey')
+    filters = {'apikey': apikey, 'denied': False}
+    key = Key.query.filter_by(**filters).first() if apikey else jwt_to_key()
```

Comment:
Ok yeah, this is the part that I overlooked and missed earlier. The `jwt_to_key()` handles a user logged in via the cookie from the backend. I'll resolve this and the other comment. Sorry about that